charts/sonarqube/values.yaml

```diff
@@ -500,3 +500,13 @@ extraConfig:
 # sonarWebContext: /
 
 terminationGracePeriodSeconds: 60
+
+# aml:
```

Comment:
the term used to be ALM as in application live-cycle management, but is now going in the application as "DevOps Platforms"